cms/src/components/content/EditContentValidation.vue

```diff
@@ -32,6 +36,12 @@ const sortedLanguages = computed(() => {
 });
 
 const showDeleteModal = ref(false);
+
+const canDeleteLang = computed(() => {
+    if (!editableContent.value) return false;
+    return verifyAccess(editableContent.value.memberOf, DocType.Language, AclPermission.Delete, "all");
```

Comment:
Think this should be:
```suggestion
    return verifyAccess(editableContent.value.memberOf, editableContent.value.parentType, AclPermission.Delete, "all");
```